app/routes/category.$categorySlug/route.tsx

```diff
@@ -1,5 +1,6 @@
 import classNames from 'classnames';
 import { LinksFunction, LoaderFunctionArgs, MetaFunction } from '@remix-run/node';
+import { products as wixStoresProducts } from '@wix/stores';
```

Comment:
I would even recommend importing only InventoryStatus

`import {products: {InventoryStatus}} from "..."` if that is possible

Reply:
not possible